Properties/AssemblyInfo.cs

```diff
@@ -56,8 +56,8 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion(Version.szBaseVersion)]
-[assembly: AssemblyFileVersion(Version.szBaseVersion)]
+[assembly: AssemblyVersion("1.0.0.0")]

```

Comment:
Please revert back to Version.* strings. This is important for the "official" builds.